src/Symfony/Component/HttpFoundation/Tests/Session/Storage/NativeSessionStorageTest.php

```diff
@@ -243,6 +243,22 @@ public function testStarted()
         $storage->start();
     }
 
+    public function testSessionCookieOnlySentOnce()
+    {
+        if (!function_exists('xdebug_get_headers')) {
+            $this->markTestSkipped('Test utilizes missing xdebug_get_headers() function.');
```

Comment:
I'm not exactly sure how to test this without a utility function, `headers_list()` doesn't work at the CLI. Inspiration from https://stackoverflow.com/a/39892373

If this is unacceptable, it at least demonstrates the test logic that needs to run.